src/pages/data-fetching.tsx

```diff
@@ -3,12 +3,12 @@ import { Heading } from 'components/Heading'
 import { Card } from 'components/Card'
 import { Text } from 'components/Text'
 import { Table } from 'components/Table'
-import { User } from 'types/schema'
 import { Badge } from 'components/Badge'
-import { useFetchUsers } from 'hooks/data/useFetchUsers'
+import { useGetUsersList, User } from 'api'
```

Comment:
Looks like we need to update the env variable in production?

![image](https://github.com/dwarvesf/nextjs-boilerplate/assets/12707960/4b4365c7-1a32-4474-ac19-a4f298c438b0)